cluster/ssl_config_test.go

```diff
@@ -58,11 +58,22 @@ func TestSSLConfig_AddClientCertAndKeyPath(t *testing.T) {
 	require.Error(t, err)
 	require.Equal(t, len(sslConfig.TLSConfig().Certificates), 0)
 	// valid client certificate, key pair
-	err = sslConfig.AddClientCertAndKeyPath("testdata/client1-cert.pem", "testdata/client1-key.pem")
+	err = sslConfig.AddClientCertAndKeyPath("../testdata/OpenSSL/client/client.crt", "../testdata/OpenSSL/client/client.key")
 	require.NoError(t, err)
 	require.Equal(t, len(sslConfig.TLSConfig().Certificates), 1)
-	// another valid client certificate, key pair
-	err = sslConfig.AddClientCertAndKeyPath("testdata/client2-cert.pem", "testdata/client2-key.pem")
+}
+
+func TestSSLConfig_AddClientCertAndEncryptedKeyPath(t *testing.T) {
+	it.MarkFlaky(t, "PKCS12 fails while decrypting")
```

Comment:
This is not a flaky test. Could you instead use something like:
```
t.Skipf("PKCS12 encrypted certificates are not yet supported")
```

Reply:
Changed, [c6481ec](https://github.com/hazelcast/hazelcast-go-client/pull/859/commits/c6481ec04328cf044ab3bec8e31bd712bd663bd2).